.github/quarkus-ecosystem-test

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+# update the version
+mvn versions:set-property -Dproperty=quarkus.version -DnewVersion=999-SNAPSHOT
```

Comment:
BTW, adding `-pl .` will apply it only to the root pom.xml, which is where it is.

Reply:
Done.